### ISSUE TITLE
Using CurrentLocationLayer on web throws "getServiceStatusStream() has not been implemented." from Geolocator

### ISSUE THREAD

**Issue by nrs1022** [tlserver/flutter_map_location_marker#101]
Hi! Using CurrentLocationLayer works fine under Android and iOS, but in web fails throwing the following error on browser's console:

Uncaught : UnimplementedError: getServiceStatusStream() has not been implemented.

It's because Geolocator doesn't implement it on web 

https://github.com/Baseflow/flutter-geolocator/issues/764#issuecomment-953975618

Is there any way to workaround it on web?

Thanks in advance.

Packages:

  flutter_map_location_marker: ^8.0.4
  geolocator: ^10.1.0

**Comment by tlserver** [comment]
I've successfully reproduced the issue with the current version (v8.0.4). As a temporary workaround while this issue is being addressed, I recommend reverting to version 8.0.2, which does not exhibit this problem.

**Comment by nrs1022** [comment]
Hi @tlserver, I've confirmed downgrading to 8.0.2 works fine because is not affected by the issue. By the way, thank you very much to address this issue, and great job with the library!

**Comment by jithware** [comment]
Have same issue on web: 

```
UnimplementedError: getServiceStatusStream() has not been implemented.
```
pubspec.yaml:
```
  flutter_map_location_marker: ^8.0.4
  geolocator: ^10.1.0
```

**Comment by tlserver** [comment]
Fixed in v8.0.5.
